flask_mongoengine/wtf/fields.py

```diff
@@ -156,7 +156,7 @@ def process_formdata(self, valuelist):
             self.data = None
 
     def _value(self):
-        return text_type(self.data) if self.data else None
+        return unicode(self.data) if self.data else None
```

Comment:
That'd break on py3. Looks like upstream is using `six` anyway, so I'd be fine just pulling it in.

/cc @mpessas

Reply:
It would, but there's already other places in this file using `unicode`. Should I update that usage as well?

Reply:
We'll have to do it eventually anyway, so fixing other calls would be cool.

Unless we're almost ready to move to upstream @wojcikstefan ?